tests/unit/test_wkt.py

```diff
@@ -167,6 +167,23 @@ def test_wkt_point_missing_values():
     assert issue_type(issues) == "invalid WKT"
 
 
+def test_wkt_geojson():
```

Comment:
Its nice to see a test here, I'm not immediately seeing the connection between the change to `parse_wkt` and the test of `wkt.normalize` though. Is it legit to just test the parse function directly?

Reply:
parse_wkt() function returns Multipolygon object which I am not sure how can be compared with expected value.

normalise() function uses parse_wkt() and processes the Multipolygon object returned by it(i.e. sets precision, checks if its valid etc). 
It returns a string type to which I have compared the expected value.

Reply:
Ok. 

Maybe testing for the change in sematics from throwing an exception to returning `(geometry, "invalid type geojson")` might have been sufficient but LGTM :-)